lib/client/client-google-pubsub.ts

```diff
@@ -142,7 +142,7 @@ export class ClientGooglePubSub extends ClientProxy {
 
     public async *getMessageIterator(
         subscription: string | Subscription,
-    ): AsyncGenerator<GooglePubSubMessage> {
+    ): AsyncGenerator<GooglePubSubMessage[]> {
         const subObj = this.getSubscription(subscription);
         for await (const message of on(subObj, GOOGLE_PUBSUB_SUBSCRIPTION_MESSAGE_EVENT)) {
             yield message;
```

Comment:
I'm way out of my element here, but if `message` is actually an array, couldn't you just loop over it and `yield` each element? Wouldn't that be an easier solution w/ a cleaner interface? If I'm using a generator, I kind of expect elements to arrive one at a time

Reply:
So we could do that, but it seems like a lot of boilerplate when we're only supposed to be yielding one message at a time. In my testing I discovered that this message is yielding an array with a single message in it. Updating the types and unpacking the array in `server-google-pubsub.ts` with a `const [message] = iterator` seemed cleaner than a nested `for` loop to me.